scripts/gen_cert.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
This script and openssl conf files are already added in scripts/tls directory.

Reply:
Thank you. I have removed the files.